### DISCUSSION TITLE
posts/Go/generics/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#289] [Announcements]
https://www.liwenzhou.com/posts/Go/generics/ 

 李文周的Blog提供免费的全套Go语言学习教程。本文主要介绍了Go语言中的泛型。Go 1.18版本增加了对泛型的支持，泛型也是目前 Go 语言开源以后所做的最大改变。

**Comment by WenLY1** [comment]
666

**Comment by seabra2** [comment]
未免有些太过于复杂了
